notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb

```diff
@@ -830,13 +830,22 @@
         "    !  gcloud builds submit --timeout=1800s --region={REGION} --tag $TRAIN_IMAGE"
```

Comment:
<p>drop 'also'</p><p>with gcloud command -&gt; with the gcloud command</p><p>enableDashboardAccess. enableWebAccess&nbsp; -&gt; enableDashboardAccess. The enableWebAccess&nbsp;</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1565/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>